slurm/roles/common/handlers/main.yml

```diff
@@ -0,0 +1,20 @@
+---
+- name: restart ntpd
+  service: name=ntpd state=restarted enabled=yes
```

Comment:
Let's be sure we use the Ansible long format for all plays.

Reply:
will do.